be/src/vec/functions/array/function_array_exists.cpp

```diff
@@ -0,0 +1,97 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include <vec/columns/column_array.h>
+#include <vec/columns/column_nullable.h>
+#include <vec/columns/columns_number.h>
+#include <vec/data_types/data_type_array.h>
+#include <vec/data_types/data_type_number.h>
+#include <vec/functions/function.h>
+#include <vec/functions/function_helpers.h>
+#include <vec/functions/simple_function_factory.h>
+
+#include <vec/utils/util.hpp>
+
+namespace doris::vectorized {
+
+// array_exists([1, 2, 3, 0]) -> [1, 1, 1, 0]
+class FunctionArrayExists : public IFunction {
+public:
+    static constexpr auto name = "array_exists";
+
+    static FunctionPtr create() { return std::make_shared<FunctionArrayExists>(); }
+
+    String get_name() const override { return name; }
+
+    size_t get_number_of_arguments() const override { return 1; }
```

Comment:
does 'array_exists' function just has one argument? From the docs, I have see multi arguments.

Reply:
Because   in fe   array_exists( lambda,  array1, array2... )  mapped to   array_exists(array_map( lambda,  array1, array2..)), 
so in be we just implement  array_exists(array).

Reply:
> does 'array_exists' function just has one argument? From the docs, I have see multi arguments.

Because in fe array_exists( lambda, array1, array2... ) mapped to array_exists(array_map( lambda, array1, array2..)),
so in be we just implement array_exists(array).